src/Service/Gateway.php

```diff
@@ -62,6 +62,7 @@ public function pay(
             $pre_order = $this->processOrder($transaction,$salesChannelContext);
             $order = $this->ginger->createOrder($pre_order);
         } catch (\Exception $e) {
```

Comment:
There no breakpoint with print a custom message about an error while creating the order using the Afterpay payment method.
I tried to pay using Afterpay and get the 'Symphony error message'

Reply:
Done